README.md

```diff
@@ -302,6 +316,25 @@ We expect the credentials for the hammer commands to to be stored inside the ham
   :password: 'changeme'
 ```
 
+## Metadata
+
+A set of data that describes and gives information about any definition.
+
+You can describe a definition using following methods available in metadata:
+
+  * **label** - specify an unique name per definition
+  * **tags** -  comma separated labels attached for the purpose of creating
+    groups of definitions
+  * **description** - specify short description about definition
+  * **param** - declare parameters for a definition using this method
+  * **for_feature** - specify feature name for a definition
```

Comment:
I would add also something like: `it implicitly confines presence of the feature`